NOLF2/ClientShellDLL/ClientShellShared/InterfaceResMgr.h

```diff
@@ -128,6 +171,33 @@ class CInterfaceResMgr
 
 #define TERMSHAREDSURF(surf) if(surf) { g_pInterfaceResMgr->FreeSharedSurface(surf); surf = NULL; }
 
+inline int CInterfaceResMgr::Get4x3Offset(int w = 0, int h = 0)
+{
+	LTFLOAT width = (LTFLOAT)GetScreenWidth();
+	LTFLOAT height = (LTFLOAT)GetScreenHeight();
+	LTFLOAT ratio = Get4x3Ratio();
+
+	if (w) {
+		width = (LTFLOAT)w;
+	}
+
+	if (h) {
+		height = (LTFLOAT)h;
+	}
+
+	return (int)(0.5f * ( width - ( height * ratio ) ));
+}
+
+//
+// I hate this name, but it's the only thing that came to mind.
+// All the interfaces seem to be designed for 640x480 then scaled up.
+//
+inline int CInterfaceResMgr::Get640x480Offset() const
```

Comment:
this can only be const if `this != g_pInterfaceResMgr` 
which it is, from the constructor.

because `Get4x3Offset()` can update  `m_dwScreenWidth` and `m_dwScreenHeight` 
in calls to `ScreenDimsChanged()`

Reply:
Done.